jquery.flot.js

```diff
@@ -1671,10 +1677,7 @@ Licensed under the MIT license.
                 point2 = axis.c2p(canvas2),
                 precision = computeValuePrecision(point1, point2, axis.direction, 1);
 
-            if (precision < 20) {
-                return precision;
-            }
-            return 20;
+            return precision;
```

Comment:
The problem with precision bigger then 20 was actually for logaxis toFixed function. Now it is checked and recomputed there.